src/Proto.Cluster/Cluster.cs

```diff
@@ -33,9 +35,9 @@ public Cluster(ActorSystem system, Serialization serialization)
             PidCache = new PidCache(this);
         }
 
-        internal Partition Partition { get; }
-        internal MemberList MemberList { get; }
-        internal PidCache PidCache { get; }
+        public Partition Partition { get; }

```

Comment:
This isn't required, I think I exposed these properties for monitoring purposes.